src/main/java/com/amihaiemil/docker/RtSwarm.java

```diff
@@ -61,4 +68,36 @@
     public JsonObject inspect() throws IOException {
         return new Inspection(this.client, this.baseUri.toString());
     }
+
+    @Override
+    public String init(final String listenAddress) throws IOException {
```

Comment:
@llorllale Why return ``String`` here and bellow? Isn't the response a Json object?